pkg/reconciler/pingsource/controller/pingsource.go

```diff
@@ -233,37 +233,24 @@ func (r *Reconciler) reconcileJobRunner(ctx context.Context, source *v1alpha1.Pi
 		return nil, err
 	}
 
-	args := resources.JobRunnerArgs{
-		ServiceAccountName: jobRunnerName,
-		JobRunnerName:      jobRunnerName,
-		JobRunnerNamespace: system.Namespace(),
-		Image:              r.jobRunnerImage,
-	}
-	expected := resources.MakeJobRunner(args)
-
 	d, err := r.deploymentLister.Deployments(system.Namespace()).Get(jobRunnerName)
 	if err != nil {
 		if apierrors.IsNotFound(err) {
-			d, err := r.kubeClientSet.AppsV1().Deployments(system.Namespace()).Create(expected)
-			if err != nil {
-				controller.GetEventRecorder(ctx).Eventf(source, corev1.EventTypeWarning, pingSourceDeploymentCreated, "Cluster-scoped deployment not created (%v)", err)
-				return nil, err
-			}
-			controller.GetEventRecorder(ctx).Event(source, corev1.EventTypeNormal, pingSourceDeploymentCreated, "Cluster-scoped deployment created")
-			return d, nil
+			return nil, fmt.Errorf("pingsource job runner deployment doesn't exist! %v", err)
+		} else {
+			return nil, fmt.Errorf("error getting job runner deployment %v", err)
 		}
-		return nil, fmt.Errorf("error getting job runner deployment %v", err)
-	} else if podSpecChanged(d.Spec.Template.Spec, expected.Spec.Template.Spec) {
-		d.Spec.Template.Spec = expected.Spec.Template.Spec
+	} else if *d.Spec.Replicas != 1 {
```

Comment:
Prev approach: create the deployment for the job runner
Now: scale up the already created deployment